src/panels/config/logs/error-log-card.ts

```diff
@@ -554,6 +578,13 @@ class ErrorLogCard extends LitElement {
       if (err.name === "AbortError") {
         return;
       }
+
+      // The stream can fail if the connection is lost or firefox service worker intercept the connection
+      if (!retry && streamLogs) {
```

Comment:
This can now also cause an infinite loop I guess? Can we make this check more specific?

Reply:
There shoulnd't be. Retry is set to false when the first chunk rendered successfully so we say we are in a healty state again. If it fails again beforehand we show the error.